### PR TITLE
server: pkiclient refetch according to voting scheule

### PR DESCRIPTION
**Pull request by mixmasala**
adjust the pki wakeup timer so that it sleeps until the pki publication deadline

### CODE REVIEW

**Comment by mixmasala** [comment]
seems to reliably fail the client with ErrNoDocument

**Comment by mixmasala** [comment]
> seems to reliably fail the client with ErrNoDocument

I forgot to wake up the pkiclient in time to publish the mix descriptor, if applicable. fixed in ddfa676

File: server/internal/pki/pki.go [hunk 1]
```diff
@@ -220,7 +220,32 @@ func (p *pki) worker() {
 			}
 		}
```
Comment (david415) [inline server/internal/pki/pki.go:222]: it's not ideal to make this worker method even longer

**Review by david415 — APPROVED** [review]
lgtm

File: server/internal/pki/pki.go (hunk 1 above)
Comment (mixmasala) [reply server/internal/pki/pki.go:222]: refactored into a method

**Comment by mixmasala** [comment]
I made one change which was to move the timer update into a method per feedback in review. Thanks!

**Pull request merged**